src/client/interfaces/IRoomInfo.ts

```diff
@@ -0,0 +1,17 @@
+import { IUserInfo } from './IUserInfo';
+
+export interface IRoomInfo {
```

Comment:
Let's call these something client side specific, otherwise developers working on the backend/server side could easily get confused as to which one to use.

Reply:
Great practice! Learned a lot 😃